plugin/AddressDetail.php

```diff
@@ -56,6 +57,11 @@ public function loadSingleAddress($id)
         if($this->getDataViewAddress() instanceof DataAddressDetailView &&
             $this->getDataViewAddress()->getPictureTypes() != [] && $this->getDataViewAddress()->getPictureTypes() != null)
             array_push($fields, "imageUrl");
+
+		if($this->getDataViewAddress() instanceof DataAddressDetailView && $this->getDataViewAddress()->getPictureTypes() != null && 
+			in_array(ImageTypes::BILDWEBSEITE, $this->getDataViewAddress()->getPictureTypes()))
+			array_push($fields, ImageTypes::BILDWEBSEITE);
+	
```

Comment:
What about combining both If conditions?

Reply:
@andernath You're right, we should combine them. I've made the changes. Thank you!